build/config/win/visual_studio_version.gni

```diff
@@ -15,36 +15,14 @@ if (is_starboard) {
     if (current_os == "winuwp") {
       wdk_version = "10.0.22000.0"
     } else {
-      wdk_version = "10.0.17763.0"
-    }
-  }
-
-  # TODO(b/210151198) Visual Studio 2022 upgrade compatibility changes.
-  # This argument should be removed, and replace the defaults for the arguments
-  # that are being overriden correspondingly.
-  declare_args() {
-    use_visual_studio_2022 = getenv("VISUAL_STUDIO_VERSION") == "2022"
-  }
-
-  if (use_visual_studio_2022) {
-    visual_studio_version = "14.34.31933"
-    if (current_os != "winuwp") {
       wdk_version = "10.0.18362.0"
     }
   }
 
   if (is_docker_build) {
     _default_visual_studio_path = "C:/BuildTools"
   } else {
-    if (use_visual_studio_2022) {
-      _vis_std_year = "2022"
-    } else {
-      _vis_std_year = "2017"
-    }
-    _default_visual_studio_path = getenv("VS2022INSTALLDIR")
```

Comment:
keep this variable. On my machine i plan to move VS2022 to C:/src, tyler's machine has it on C:/Program Files/

Reply:
You can use the gn arg `visual_studio_path` below instead of the environment variable.

Reply:
Ack, Oscar's suggestion works. You can specify it like follows:
https://gn.googlesource.com/gn/+/main/docs/reference.md#buildfile-functions-declare_args_declare-build-arguments-example